src/main/default/classes/TriggerConfigService.cls

```diff
@@ -0,0 +1,115 @@
+public class TriggerConfigService {
+    public static ObjectConfig getConfig(String objName) {
+        String lowercaseObjName = objName.toLowerCase();
+        if (!configMetadata.keySet().contains(lowercaseObjName)) {
+            return null;
```

Comment:
this is the same as `configMetadata.get(lowercaseObjName);` where the key doesn't exists

Reply:
think you mean `if(!configMetadata.containsKey(lowercaseObjName)`

Reply:
kind of think it would make upstream stuff easier if we returned a base ObjectConfig

```
if (config == null) {
  config = new ObjectConfig();
  config.objName = lowerCaseObjName;
}
return config;
```

Reply:
Went back and forth on this. Feels safer if we don't leave the possibility of a null ObjectConfig 👍